.devcontainer/requirements.txt

```diff
@@ -58,7 +58,7 @@ black[jupyter]==23.7.0
     # via
     #   -r lint-requirements.txt
     #   blacken-docs
-blacken-docs==1.16.0
+blacken-docs==1.18.0
```

Comment:
note: This update is to include [this commit](https://github.com/adamchainz/blacken-docs/commit/adc7e695391be07e6a50140fd073e3a84957a0a0) that introduce the https://github.com/adamchainz/blacken-docs/blob/002ed0ed3c9e4f95ae8901744a0d91e174b3d03e/README.rst#restructuredtext) command. We need this command to bypass the issue that `black` cannot parse jupyter magic commands like `%%writefile`.